gooddata-java-model/src/main/java/com/gooddata/sdk/model/warehouse/Warehouse.java

```diff
@@ -5,16 +5,19 @@
  */
 package com.gooddata.sdk.model.warehouse;
 
-import com.fasterxml.jackson.annotation.*;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
The `annotation.*` import is pretty repeating pattern - looks like someone made it by intention. @liry do you have any opinion on this?